config/settings/local.py

```diff
@@ -1,4 +1,10 @@
 import os  # noqa
+from pathlib import Path
+
+import environ
+
+ROOT_DIR = Path(__file__).resolve(strict=True).parent.parent.parent
+environ.Env.read_env(os.path.join(ROOT_DIR, '.env'))
```

Comment:
Если я не ошибаюсь, то можно было сделать так
```suggestion
environ.Env.read_env(ROOT_DIR / '.env')
```

Идея Path в том, чтобы избавиться от os.path.